versions.props

```diff
@@ -12,7 +12,7 @@ com.googlecode.json-simple:json-simple = 1.1.1
 com.jayway.awaitility:awaitility = 1.6.5
 com.palantir.config.crypto:encrypted-config-value-module = 1.0.0
 com.palantir.remoting:* = 0.13.0
-com.palantir.remoting1:* = 1.0.3
+com.palantir.remoting1:* = 1.1.0
```

Comment:
We don't believe this will cause problems as these clients are only used for timelock service(s) communication which should be upgraded in lock step (since we don't currently support rolling upgrades).  With that said, it doesn't even look like the change would effect us at all since we also control the servers with which these clients are communicating and I believe our dropwizard services use 1.2.  This is certainly true for the timelock service, I guess we would need to confirm for the reset of the internal products...